examples/compressors_comparison.py

```diff
@@ -78,26 +78,33 @@
 # best compromise between good compression and speed.
 
 ###############################################################################
-# Start by measuring the time spent for dumping of the zlib data
+# Start by measuring the time spent for dumping of the zlib data:
+
 start = time.time()
 with open(pickle_file, 'wb') as f:
     dump(data, f, compress='zlib')
 zlib_dump_duration = time.time() - start
 
 ###############################################################################
-# Then measure the size of the zlib dump data on disk
+# Then measure the size of the zlib dump data on disk:
+
 zlib_file_size = os.stat(pickle_file).st_size / 2**20
 
 ###############################################################################
-# Finally measure the time spent for loading the zlib data. Note that the
-# compression format is detected automatically by Joblib using the standard
-# magic number present at the beginning of the file. This is the case for all
-# compression methods supported by Joblib.
+# Finally measure the time spent for loading the compressed dataset:
+
 start = time.time()
 with open(pickle_file, 'rb') as f:
     load(f)
 zlib_load_duration = time.time() - start
 
+###############################################################################
+# Note: The compression format is detected automatically by Joblib.
+#   The compression format is identified by the standard magic number present
+#   at the beginning of the file. Joblib uses this information to determine
+#   the compression method used.
+#   This is the case for all compression methods supported by Joblib.
```

Comment:
This does not seem like valid rst: the indentation after "note" is strange. But maybe I am wrong. Did you check that it renders right?

Reply:
I pushed an update with the valid rst: much better rendering indeed ! Thanks.